src/PAModelTests/PAModelTests.csproj

```diff
@@ -41,6 +41,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <None Remove="Apps\UnicodeEscapingInTestSteps.msapp" />
```

Comment:
Guess this was auto added. Might be good to remove this line

Reply:
You're right. I almost snuck it in before you found it 😄 Fixed now.